code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -58,6 +58,7 @@ obj/item/clothing/shoes/sandal/clogs
 	name = "magic shoes"
 	icon_state = "black"
 	body_parts_covered = FEET
+	origin_tech = list(TECH_BLUESPACE = 3, TECH_ARCANE = 5)

```

Comment:
I'm fairly certain these don't appear anywhere but the Technomancer ship, but given that, it's probably best to have it at max a 2. They are unique, but, aside from somehow getting OldWizard, they don't actually do anything "arcane".